.changeset/seven-crews-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Ungrouping styles in progress-bar.scss
```

Comment:
Not sure what to write in changeset, any recommendations are appreciated :)

Reply:
Maybe:
```suggestion
Separate background pseudo-class selectors in progress-bar.scss
```